addons/validate_mission/fn_run.sqf

```diff
@@ -156,6 +156,72 @@ private _check_cba_settings = {
     ["Changed CBA settings", true, [], _strings];
 };
 
+private _check_playable_units = {
+    private _msg = [];
+    private _pass = true;
+    if (count playableUnits < 35) then {
+        (_msg pushBack "Not enough playable units: 35 required")};
+        _pass = false;
+    if ((group player get3DENAttribute "a3aa_ee_persistent_callsign" select 0) isEqualTo "") then {
+        _msg pushback "Possible AI unit marked as player.";
+        _msg pushback "Did you place some AI before any playable units?";
```

Comment:
Why not re-define `_msg` here (and everywhere below) instead of tons of `pushBack`s?

Ie.
```sqf
private ["_msg"];   // or private _msg = nil;   or private _msg = [];
if (...) then {
    _msg = [
        "Possible AI unit marked as player.",
        "Did you place some AI before any playable units?"
    ];
} else {
    _msg = [
        "Whatever.",
    ];
};

["Blabla", _pass, [], _msg]
```
The parent scope is preserved.